src/bundle/Resources/public/js/scripts/helpers/notification.helper.js

```diff
@@ -60,22 +60,36 @@
      * @function showDangerNotification
      * @param {String} message
      */
-    const showDangerNotification = (message) =>
+    const showDangerNotification = (message) => {
+        console.warn('[DEPRECATED] showDangerNotification is deprecated');
+        console.warn('[DEPRECATED] it will be removed from ezplatform-admin-ui 2.0');
+        console.warn('[DEPRECATED] use showErrorNotification instead');
+
         showNotification({
```

Comment:
it should be `showErrorNotification(message)` after changes

Reply:
Fixed!